demos/blockfactory/factory.js

```diff
@@ -99,8 +99,8 @@ function updateLanguage() {
  */
 function formatJson_(blockType, rootBlock) {
   var JS = {};
-  // Type is not used by Blockly, but may be used by a loader.
```

Comment:
This PR should not contain any files in the demos directory.  With one exception...